src/screens/LocalCommunity/index.js

```diff
@@ -144,6 +162,17 @@ const LocalCommunity = () => {
     await setLocation(tempLocation);
     await setLocationPost(returnTempLocation);
     await setLocationLog(locLog);
+
+    if (listIndex === 0)
+      return AnalyticsEventTracking.eventTrack(
+        BetterSocialEventTracking.ONBOARDING_LOCATION_FIRST_DELETED
+      );
+
+    if (listIndex === 1) {
+      return AnalyticsEventTracking.eventTrack(
+        BetterSocialEventTracking.ONBOARDING_LOCATION_SECOND_DELETED
+      );
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, this code only tracks the deletion of the first and second locations. If it's necessary to track the deletion of all locations, consider adding an additional condition for that.
```diff
+    if (listIndex >= 2) {
+      return AnalyticsEventTracking.eventTrack(
+        BetterSocialEventTracking.ONBOARDING_LOCATION_NTH_DELETED
+      );
+    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->